iocore/net/test_certlookup.cc

```diff
@@ -206,7 +206,8 @@ SSLReleaseContext(SSL_CTX *ctx)
 int
 main(int argc, const char **argv)
 {
-  diags = new Diags(NULL, NULL, stdout);
+  BaseLogFile *blf = new BaseLogFile("stdout");
+  diags = new Diags(NULL, NULL, blf);
```

Comment:
If the third parameter is changed into `const char *`, there will exist constructor ambiguities when, for example, `NULL` is the third parameter. I'm probably missing something here, but I think any changes (that I can currently think of) made to the API to disambiguate the parameters would decrease the overall understandability of the API.